### ISSUE TITLE
How to add divalent cations?

### ISSUE THREAD

**Issue by mce13** [lequieu/1cpn-model#10]
Hi there,

Is there a way to add some divalent cations, preferably Mg2+?

Cheers,

mce

**Comment by yihengwuKP** [comment]
I don't think so. Originally, with 3spn, no explicit ion model for divalent ions like Mg2+ was developed. As a result, it is not straightforward to introduce Mg2+ in 1cpn. Depending on your use case, there might be ways to remedy this: 1. If the expected effect of Mg2+ is simply reducing the persistence length of DNA, then this might be incorporated through tuning the angle potential of the DNA chain. 2. If you want to study the Mg2+ binding with phosphate, etc, then there is no easy way of doing this for now. @lequieu feel free to add anything or correct me if I am wrong.
